docs/CONTRIBUTING.md

```diff
@@ -7,6 +7,11 @@ git clone git@github.com:kinvolk/lokomotive.git
 cd lokomotive
 ```
 
+## Requirements
+
+- Minimum Go version **1.4**
```

Comment:
It seems to me, that minimal version of Go we need is 1.12, as building with 1.11 gives following compile errors:
```
CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build \
        -mod=vendor \
        -ldflags "-X github.com/kinvolk/lokomotive/pkg/version.Version=`git describe --tags --always`-dirty -extldflags '-static'" \
        -buildmode=exe \
        -o lokoctl \
        github.com/kinvolk/lokomotive/cmd/lokoctl
# github.com/docker/cli/cli/config
vendor/github.com/docker/cli/cli/config/config.go:109:18: undefined: os.UserHomeDir
# helm.sh/helm/v3/pkg/chart/loader
vendor/helm.sh/helm/v3/pkg/chart/loader/archive.go:145:7: undefined: strings.ReplaceAll
# helm.sh/helm/v3/pkg/kube
vendor/helm.sh/helm/v3/pkg/kube/client.go:525:21: undefined: strings.ReplaceAll
make: *** [Makefile:52: build-slim] Error 2
make: *** [Makefile:35: build-in-docker] Error 2
```

Reply:
@invidian I tried with `go1.13.5` and method `t.Cleanup` was undefined. I got to find it from [here](https://ieftimov.com/post/testing-in-go-clean-tests-using-t-cleanup/)

Reply:
Ah yes, hehe. I didn't run the tests :) I'm all in favor for requiring 1.14 then :+1: